styles/_utilities.scss

```diff
@@ -1,7 +1,7 @@
 .underline-link {
-    @include font-size(s);
+    font-size: 22px;
```

Comment:
**Question**: As I understand the `font-size` mixin (`@font-size(m)`) would be set a `22px` on a `medium` size display (media query), but set to `20px` on small displays and `24px` on large. Is there any reason we'd want to avoid using `@font-size(m)` here?

Reply:
I'm being pedantic and following the design precisely, as in the design this link is a font size `22px` on a large display and not `24px`. Having the link slightly larger than expected on mobile works well in my opinion, Ben Enright might  flag this in design review for correction. I will make a note to ask him about this.